care/abdm/utils/fhir.py

```diff
@@ -190,12 +190,12 @@ def _procedure(self, procedure):
                 text=procedure["procedure"],
             ),
             subject=self._reference(self._patient()),
-            performedDateTime=f"{procedure['time']}:00+05:30"
-            if not procedure["repetitive"]
-            else None,
-            performedString=f"Every {procedure['frequency']}"
-            if procedure["repetitive"]
-            else None,
+            performedDateTime=(
+                f"{procedure['time']}:00+05:30" if not procedure["repetitive"] else None
+            ),
+            performedString=(
+                f"Every {procedure['frequency']}" if procedure["repetitive"] else None
+            ),
```

Comment:
@DraKen0009 could you quickly review the files changed in the PR? I think these changes were not intended to be present in this PR.

Reply:
@rithviknishad  this file updated by the pre-commits formatting.  I had to update a reference of `spo2` to `archieved_spo2` in the file after which this file got formatted as well